### DISCUSSION TITLE
GitHub Enterprise Cloud - Data Residency

### DISCUSSION THREAD

**Discussion by khaalam-moj** [community/community#134008] [Enterprise]
### Select Topic Area

Question

### Body

Hi,
Can someone please provide a system overview/high level architecture of GitHub Enterprise Cloud?
Data management (storage, replication etc)
Underlying infrastructure etc.

**Comment by queenofcorgis** [comment]
@khaalam-moj are you currently an Enterprise customer and/or admin?
